cove/views.py

```diff
@@ -60,8 +60,45 @@ def get_releases_aggregates(json_data):
 def get_grants_aggregates(json_data):
     count = len(json_data['grants']) if 'grants' in json_data else 0
     
+    ids = []
+    unique_ids = []
+    #release_dates = []
+    #earliest_release_date = None
+    #latest_release_date = None
+    table_data = {}
+    if 'grants' in json_data:
+        for grant in json_data['grants']:
+            # Gather all the ocids
+            ids.append(grant['id']) if 'id' in grant else 0
+            
+            #Gather all the release dates
+            #release_dates.append(release['date']) if 'date' in release else 0
+            
+            # Some identifying data from a release that could be displayed in a table
+            generic_info = {}
+            if 'title' in grant:
+                generic_info['title'] = grant['title']
```

Comment:
Line 79 and 80 are maybe clearer written as:

```
generic_info['title'] = grant.get('title')
```

This is also true of the next two if statements (but not line 85).